src/NHibernate.Test/Async/StaticProxyTest/InterfaceHandling/Fixture.cs

```diff
@@ -179,23 +179,23 @@ public async Task ProxyInterfaceIdAccessFromDifferentInterfacesAsync()
 
 		private void ThrowOnIEntityNameAccess(IEntity entity)
 		{
-			Assert.That(() => entity.Name, Throws.TypeOf<ObjectNotFoundException>(), "IEntity.Name access should lead to proxy initialization");
+			Assert.That<string>(() => entity.Name, Throws.TypeOf<ObjectNotFoundException>(), "IEntity.Name access should lead to proxy initialization");
```

Comment:
Why such changes are necessary?

Reply:
Because .NET 6 broke [NUnit](https://github.com/nunit/nunit/issues/3993).

Reply:
Probably need to wait next 6.0.x SDK version before merging.

Reply:
Yeah.. Quite annoying bug...

Reply:
Fixed in 6.0.101